docs/build/training.md

```diff
@@ -69,12 +108,19 @@ These dependency versions should reflect what is in the [Dockerfiles](https://gi
     export MPI_CXX_INCLUDE_PATH=<location for openmpi/include/>
     ```
 
-3. Create the ONNX Runtime wheel
+3. Create the ONNX Runtime Python wheel
+
+   ```bash
+   ./build.sh --config=RelWithDebInfo --enable_training --build_wheel --use_cuda --cuda_home {location of cuda libs eg. /usr/local/cuda/} --cudnn_home {location of cudnn libs eg./usr/local/cuda/} --cuda_version={version for eg. 11.8} --mpi_home <location for openmpi eg. /usr/local/include/>
```

Comment:
I think we can avoid having mpi related in the sample, as it is not used for real usage except for the legacy orttrainer users. Now by default, training don't requires mpi/nccl unless we enable with onnxruntime_USE_MPI/onnxruntime_USE_NCCL.

Reply:
I marked it as optional earlier in this doc. But I think you're right, we should not need to ask users to install mpi/nccl as the distributed stuff is done outside of ort as of now.

Will update the doc.